apps/jobs/urls.py

```diff
@@ -9,6 +9,10 @@
     url(r'challenge/(?P<challenge_id>[0-9]+)/'
```

Comment:
The title of the PR says "api to show number of remaining submissions" but it also involves changes related to other apis also. Most of the changes are because of running autopep8. Can you please remove those changes so that this PR is only concerned with remaining submission api?